services/ui-src/src/utils/forms/forms.ts

```diff
@@ -83,12 +82,27 @@ export const initializeChoiceFields = (fields: FormField[]) => {
       // initialize choice as controlled component in unchecked state
       if (choice.checked != true) choice.checked = false;
       // if choice has children, recurse
-      if (choice.children) initializeChoiceFields(choice.children);
+      if (choice.children) initializeChoiceListFields(choice.children);
     });
   });
   return fields;
 };
 
+// add initial blank option to dropdown fields if needed
+export const initializeDropdownFields = (fields: FormField[]) => {
+  const dropdownFields = fields.filter(
+    (field: FormField) => field.type === "dropdown"
+  );
+  dropdownFields.forEach((field: FormField) => {
+    // if first option is not already a blank default value
+    if (field?.props?.options[0].value !== "") {
+      // add initial blank option
+      field?.props?.options.splice(0, 0, { label: "", value: "" });
```

Comment:
Would we want to add `- Select an Option -` as the initial "blank" option as per the [CMS Dropdown](https://design.cms.gov/components/dropdown?theme=core)?

Reply:
Awesome question. For right now we've opted for blank option to avoid a hydration value flicker, but this is something for us to talk to design about for sure. 👍

Reply:
@karla-vm @ntsummers1 

Fixed here! https://github.com/CMSgov/mdct-mcr/pull/3742 🎉 

Thanks for the solid review comment.